getting-started/scaling-out.md

```diff
@@ -0,0 +1,181 @@
+  TimescaleDB extension in it.
+  
+* Ensure that the local user has `USAGE` privilege on the `timescaledb_fdw` 
+  foreign data wrapper on the access node.
+  
+```sql
+SELECT add_data_node('node1', host => 'dn1.example.com',
+  password=>'<remote_password>', bootstrap_user=>'<superuser>',
+  bootstrap_password=>'<superuser_password>');
+SELECT add_data_node('node2', host => 'dn2.example.com',
+  password=>'<remote_password>', bootstrap_user=>'<superuser>',
+  bootstrap_password=>'<superuser_password>');
+```
+
+Any additional users that will access a distributed hypertable currently 
+need their own user mappings per data node with a `user` and `password` option. 
+A user mapping can be created for a data node as follows: 
+
+```sql
+CREATE USER MAPPING FOR <another_user> SERVER node1 
+  OPTIONS (user '<remote_user>' password '<remote_password>');
+```
+The additional users also need `USAGE` permission on the `timescaledb_fdw` 
+foreign data wrapper and any data node (server) objects.
+
+Deleting a data node is done by calling `delete_data_node`:
+
+```sql
+SELECT delete_data_node('node1');
+```
+>:TIP: Note that a data node cannot be deleted if it contains data for a
+hypertable, since otherwise data would be lost.
+
+### Information Schema for Data Nodes
+
+The data nodes that have been added to the distributed database 
+can be found by querying the
+[`timescaledb_information.data_node`][timescaledb_information-data_node] view.
+
+## Working with Distributed Hypertables
+
+As previously mentioned, distributed hypertables are similar to normal
+hypertables with the difference that distributed hypertables will
+partition the data and store it on remote data nodes. This means that
+to create a distributed hypertable, you need to have added at least one data node
+before starting.
+
+### Creating Distributed Hypertables
+
+As with normal hypertables, start by creating a regular
+table to hold the data, then transform it into a distributed
+hypertable. In contrast to when creating a normal hypertable, we use
+the function
+[`create_distributed_hypertable`][create_distributed_hypertable].
+
+```sql
+CREATE TABLE conditions (
+  time        TIMESTAMPTZ       NOT NULL,
+  location    TEXT              NOT NULL,
+  temperature DOUBLE PRECISION  NULL,
+  humidity    DOUBLE PRECISION  NULL
+);
+
+SELECT create_distributed_hypertable('conditions', 'time', 'location');
+```
+
+>:WARNING: If there are no data nodes available, you will get an error
+>when executing `create_distributed_hypertable` and the distributed
+>hypertable will not be created.
+
+This creates a multi-dimensional distributed hypertable 
+partitioned along `time` and `location`.
+You can now insert data into the distributed hypertable and
+it will automatically be partitioned on the available data nodes
+by using the provided space partition. You
+can find more information for how to work with data in hypertables in the
+section [Creating Hypertables][creating-hypertables].
```

Comment:
Should this link to "Using hypertables"?

Reply:
The doc is called Creating hypertables.